lib/plant.py

```diff
@@ -1,4 +1,23 @@
+import time
+import board
+import busio
+from adafruit_seesaw.seesaw import Seesaw
+
 class Plant():
 
     def soil_moisture(self):
-        return 350
+
+        i2c_bus = busio.I2C(board.SCL, board.SDA)
+
+        ss = Seesaw(i2c_bus, addr=0x36)
+
+        while True:
+            moisture_level = ss.moisture_read()
+            # temp = ss.get_temp()
+            try:
```

Comment:
Do you need an except: clause here?

Reply:
Think we could add an except: clause here. May be a good idea.